### DISCUSSION TITLE
Display people's company using MS Search

### DISCUSSION THREAD

**Discussion by MichaelJata** [microsoft-search/pnp-modern-search#2071] [Q&A] [answered]
I am using v. 4.6 and trying to build a people directory showing the users' companies. I understood that using the MS Search instead of SharePoint Search, the Graph API with the entity "person" is used. So according to the following documentation, "companyName" should be available:
https://docs.microsoft.com/en-us/graph/api/resources/person?view=graph-rest-beta

The company name attribute in AAD is filled but when adding it to the properties ("companyName") of the search result web part, it is not displayed at all ( I am checking with the debug layout).

Am I doing/expecting something wrong?
Any hint would be appreciated

**Comment by wobba** [comment, marked as answer]
This seems to be an API issue, not a web part issue. Try out your queries using the Graph Explorer. If you can repro, then log the issue with Graph.

**Comment by MichaelJata** [comment]
@wobba: You seem to be absolutely right.
I found that the Graph API's endpoint /me/people is capable of returning the companyName while the /search/query is not. Since the latter is used in the web part, it won't show up.
Thanks, I will log the issue as you suggested.

![image](https://user-images.githubusercontent.com/1466196/168079688-f818284c-8445-4d8c-bdf9-5f52e183c4a2.png)

_________________ 
![image](https://user-images.githubusercontent.com/1466196/168081049-5a6117fc-a098-491d-bb16-a3082c55c4d4.png)
